### DISCUSSION TITLE
Windows 11 24H2: How can I resize the settings button flyout?

### DISCUSSION THREAD

**Discussion by bege10** [ramensoftware/windhawk-mods#1091] [Q&A]
After the update to Windows 11 24H2 the flyout for the settings button on the Windows 10 ep task bar is narrower than before and shows only 2 lines of buttons. To reach the other buttons one must scroll. 
Is it possible to give the flyout the full size it had with 23H2?

![Screenshot 2024-10-16 093624](https://github.com/user-attachments/assets/0d78f9c8-a71d-4693-9b13-cbc96c041b14)

**Comment by m417z** [comment]
It might be possible with the [Windows 11 Notification Center Styler](https://windhawk.net/mods/windows-11-notification-center-styler) Windhawk mod. You can refer to the guide for examples, or try creating an issue there:
https://github.com/ramensoftware/windows-11-notification-center-styling-guide

**Reply by bege10** [reply]
> It might be possible with the [Windows 11 Notification Center Styler](https://windhawk.net/mods/windows-11-notification-center-styler) Windhawk mod. 

How can I examine a flyout of the ExplorerPatcher Windows 10 sys tray With UWPspy? Ctrl+D does not work because UWPspy does not have focus when the flyout is shown. Or is there the same issue that explorer.exe cannot be examined with the Windows 10 tasbar?

**Reply by m417z** [reply]
Target `ShellExperienceHost.exe`.

**Reply by AJolly** [reply]
ps - @bege10 This reg file will stop the window from automatically closing, so it'll be easier for you to use Ctrl+D

[HKEY_LOCAL_MACHINE\SOFTWARE\Microsoft\Windows\CurrentVersion\ImmersiveShell\Launcher]
"DisableLightDismiss"=dword:00000001

**Reply by bege10** [reply]
> ps - @bege10 This reg file will stop the window from automatically closing, so it'll be easier for you to use Ctrl+D
> 
> [HKEY_LOCAL_MACHINE\SOFTWARE\Microsoft\Windows\CurrentVersion\ImmersiveShell\Launcher] "DisableLightDismiss"=dword:00000001

Thank you, that helps to examine the flyout.
Unfortunately I am not skilled enough to find and change the right items. I will live with the current situation.
